libcxx/utils/ci/buildkite-pipeline.yml

```diff
@@ -1135,3 +1135,33 @@ steps:
           - exit_status: -1  # Agent was lost
```

Comment:
We should add the support mention in the documentation in this patch.

Reply:
We accidentally already did that in another change. 

We can go ahead and revert that documentation change and move it here, if you would like? I suggested @rprichard didn't do that yesterday because I believed this follow up would land shortly.

Reply:
Oh, that documentation change was already reverted in https://github.com/llvm/llvm-project/pull/69660.